user/wl-kbptr/template.py

```diff
@@ -0,0 +1,22 @@
+pkgname = "wl-kbptr"
+pkgver = "0.2.0"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = ["meson", "pkgconf"]
+makedepends = [
+    "cairo-devel",
+    "libxkbcommon-devel",
+    "wayland-devel",
+    "wayland-protocols",
+]
+pkgdesc = "Control the mouse pointer with the keyboard"
+maintainer = "ttyyls <contact@behri.org>"
+license = "GPL-3.0-or-later"
```

Comment:
this is not actually specified whether it's -only or -or-later anywhere

Reply:
afaik the default GPL license is in use with copyright assignment to the fsf and that should be synonymous with the -or-later option

Reply:
even if it's not the case, the compatibility matrix says it's ok to relicense GPL-v3 to GPL-v3-or-later https://www.gnu.org/licenses/gpl-faq.html#compat-matrix-footnote-7